samples/client/petstore/typescript-node/default/model/index.ts

```diff
@@ -0,0 +1,19 @@
+export * from './apiResponse';
```

Comment:
is this index file still being generated?
why does the RequestDetailedFile occur in index.ts and models.ts?

Reply:
Nope, it's not being generated anymore - removed!